### DISCUSSION TITLE
Antlers form within Blade template

### DISCUSSION THREAD

**Discussion by ARCHES-Academy** [statamic/cms#6534] [Q&A] [answered]
I know there are a couple places in the docs where it talks about using [Antlers in Blade](https://statamic.dev/blade#using-tags-with-blade), but I've tried a couple variations on this and can't quite get it working. I'm sure I'm doing something wrong, but I'm unsure exactly what.

Basically I'm trying to put a [user login form](https://statamic.dev/tags/user-login_form#overview), and I know that the antlers for this is fairly simple (copied directly from the docs):

``` antlers
{{ user:login_form }}
 
    {{ if errors }}
        <div class="bg-red-300 text-white p-2">
            {{ errors }}
                {{ value }}<br>
            {{ /errors }}
        </div>
    {{ /if }}
 
    {{ if success }}
        <div class="bg-green-300 text-white p-2">
            {{ success }}<br>
        </div>
    {{ /if }}
 
    <label>Email</label>
    <input type="email" name="email" value="{{ old:email }}" />
 
    <label>Password</label>
    <input type="password" name="password" value="{{ old:password }}" />
 
    <button type="submit">Log in</button>
 
{{ /user:login_form }}
```

However the Blade syntax for Antlers doesn't seem to mesh well with this, and every variation I've attempted has resulted in a 500 Server Error.

I know it'll have some variation of `Statamic::tag('user:loginForm')` but I haven't had much success beyond that.

I tried
```php
@php (Statamic::tag('user:loginForm'))
```
and
```php
@foreach (Statamic::tag('user:loginForm'))
```
and even
```php
@php (Statamic::tag('user:loginForm')->fetch())
```
but no luck yet.

What am I missing?

**Comment by ryanmitchell** [comment]
Try again... if you check out this PR, you'll see a code example of how it works:

https://github.com/statamic/cms/pull/5892

Did you try `Statamic::tag('user:login_form')` instead of `loginForm` ?

**Reply by ARCHES-Academy** [reply]
I attempted to make sense of that, and I ended up getting a Array to String conversion server error. Here's the code I ended up with, but I'm sure I'm still doing something wrong:
```blade
        @php
            $form = Statamic::tag('user:login_form')->params([
                'redirect' => url('/staff'),
                'id' => 'login-form',
            ])->fetch();
        @endphp
        <form {!! $form['attrs'] !!}>
            <label>Email</label>
            <input type="email" name="email" value="" />

            <label>Password</label>
            <input type="password" name="password" value="" />

            <button type="submit">Log in</button>
            {!! $form['params'] !!}
            <input name="{!! $form['honeypot'] !!}">
        </form>
```

I tried emulating the code on that page you linked to, as well as the one it refrences. I'm still a beginner when it comes to PHP & Blade, but I'm trying to learn.

Thanks!

**Comment by jasonvarga** [comment, marked as answer]
You can also mix the two, if you're not opposed to that.

In your Blade view, include an Antlers partial.

```blade
@include('form')
```

```antlers
// form.antlers.html

{{ user:login_form }} ... {{ /user:login_form }}
```

**Reply by ARCHES-Academy** [reply]
You know, I hadn't considered that, but that may be the simplest option. I'll probably give that a try, thanks!

**Reply by ARCHES-Academy** [reply]
This is what I ended up doing. The form works now, and I'll probably end up reusing this idea for later project bits that are simpler in Antlers. Thanks for the help!
